app/assets/src/components/views/discovery/mapping/DiscoveryMap.jsx

```diff
@@ -51,6 +53,10 @@ class DiscoveryMap extends React.Component {
       />
     );
     this.setState({ tooltip, tooltipShouldClose: false });
+
+    logAnalyticsEvent("DiscoveryMap_marker_hovered", {
+      locationId: hoverInfo.id,
```

Comment:
nit: `hoverInfo` could be `locationInfo` to be more explicit...

Reply:
👍